portal/src/graphql/adminapi/UserDetailsConnectedIdentities.tsx

```diff
@@ -49,39 +52,41 @@ interface UserDetailsConnectedIdentitiesProps {
   availableLoginIdIdentities: string[];
 }
 
-const identityTypes = ["email", "phone", "username"] as const;
-type IdentityType = typeof identityTypes[number];
+const loginIdIdentityTypes = ["email", "phone", "username"] as const;
+type LoginIDIdentityType = typeof loginIdIdentityTypes[number];
+type IdentityType = "login_id" | "oauth";
```

Comment:
This should be defined as `"ANONYMOUS" | "LOGIN_ID" | "OAUTH"`, and the `type` field in `Identity` type should use it.

The identity list should explicitly hide (i.e. ignore) anonymous identities.